core/src/main/java/com/linecorp/armeria/common/logging/StringAggregatingPreviewer.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.logging;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+import java.nio.charset.Charset;
+
+import com.linecorp.armeria.common.HttpHeaders;
+
+import io.netty.buffer.ByteBuf;
+
+final class StringAggregatingPreviewer extends ByteBufAggregatingPreviewer {
```

Comment:
How about renaming to `TextualContentPreviewer` for consistency with `TextualContentPreviewerFactory`?